shared/common-adapters/name-with-icon/index.tsx

```diff
@@ -116,7 +116,7 @@ class NameWithIcon extends React.Component<NameWithIconProps> {
         usernames={[username]}
         colorBroken={this.props.colorBroken}
         colorFollowing={this.props.colorFollowing}
-        colorYou={this.props.notFollowingColorOverride}
+        colorYou={this.props.notFollowingColorOverride || true}
```

Comment:
????

Reply:
`Usernames` expects a string color to override your own username or `true` to set it to black. The ticket requested we change it in team building but it seemed fine to change it for `NameWithIcon`

Reply:
this makes it always true?

Reply:
This makes it so your own username in `NameWithIcon` is black by default

Reply:
Why have the parameter then